requirements.txt

```diff
@@ -0,0 +1,2 @@
+holmesv @ git+https://github.com/HelloChatterbox/holmesV
```

Comment:
this really shouldn't be in requirements, it will break things if someone installs this plugin in "vanilla mycroft"

Reply:
removed and added install to actions workflow for unit tests